android/src/com/unciv/app/PlatformSpecificHelpersAndroid.kt

```diff
@@ -34,13 +36,23 @@ Sources for Info about current orientation in case need:
     }
 
     override fun isInternetConnected(): Boolean {
```

Comment:
If you're changing this to not only check internet connectivity, there are actually 2 fail states and so A. the function name should be changed (checkConnectionToMultiplayerServer or something) and B. we should tell the user if it's a problem with his internet or the server itself, so either return a fail string or an enum or something.

This also means we can keep the already-translated line of "No internet connection!" and just add another translation line of "The server is unreachable" or something

Reply:
yeah I'm gonna split the 2 in 2 separate functions